<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DecodeTools</class>
 <widget class="QWidget" name="DecodeTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DecodeTools</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.ico</normaloff>logo.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="fileformat_cmb">
       <item>
        <property name="text">
         <string>OpenRTK330LI</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RTK330LA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>INS401</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mixed_Raw</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>IMU</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>E2E_Protocol</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RTCM_SPP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Convbin.exe</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RTCM_2_HEX</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BeiDou</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NPOS112</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>STRTCM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ublox</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filepath_edt"/>
     </item>
     <item>
      <widget class="QToolButton" name="select_btn">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="time_lb">
       <property name="text">
        <string>00:00:00:000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>kml frequency</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="frequency_cmb">
       <item>
        <property name="text">
         <string>1hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100hz</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="displayFormat">
        <string>yyyy/MM/dd H:mm:ss</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="time_checkBox">
       <property name="toolTip">
        <string comment="解码ins401的时候，第一列以年月日时分秒的时间格式显示时间" extracomment="解码ins401的时候，第一列以年月日时分秒的时间格式显示时间">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;解码ins401的时候，第一列以年月日时分秒的格式显示时间。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="statusTip">
        <string comment="解码ins401的时候，第一列以年月日时分秒的时间格式显示时间" extracomment="解码ins401的时候，第一列以年月日时分秒的时间格式显示时间"/>
       </property>
       <property name="whatsThis">
        <string comment="解码ins401的时候，第一列以年月日时分秒的时间格式显示时间" extracomment="解码ins401的时候，第一列以年月日时分秒的时间格式显示时间">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>UTC time(YYYY/MM/DD-hh:mm:ss.dd)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="decode_btn">
       <property name="text">
        <string>decode</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_setting">
       <property name="toolTip">
        <string>setting</string>
       </property>
       <property name="text">
        <string>s</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
